src/render/graph.rs

```diff
@@ -352,7 +370,7 @@ impl Graph {
         }
 
         // Return the output buffer of destination node
-        &self.nodes.get(&NodeIndex(0)).unwrap().outputs[0]
+        self.nodes.get(&NodeIndex(0)).unwrap().borrow().outputs[0].clone()
```

Comment:
I removed your `unsafe` use here just by cloning out the buffer.
In case of the online AudioContext rendering, we need an owned version anyway.
For OfflineAudioContext I think the benefit of no-unsafe is worth the `Rc` increment from `clone`